tools/old/biosnoop.py

```diff
@@ -16,7 +16,8 @@
 from bcc import BPF
 
 # load BPF program
-b = BPF(text="""
+b = BPF(
```

Comment:
We might want to ignore files under `tools/old/`. Those are just there for posterity.